<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomWidget</class>
 <widget class="QWidget" name="CustomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="colorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorButton">
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFromScreenButton">
       <property name="text">
        <string>Select Color From Screen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RGB 255</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="red255">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="green255">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="blue255">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="alpha255">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyQColorButton">
         <property name="toolTip">
          <string>Copy As QColor::fromRgb</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteQColorButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="hexLineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyHexButton">
         <property name="toolTip">
          <string>Copy as QColor::fromRgb</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHexButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>RGBF</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="redf">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000001000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="greenf">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000001000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="bluef">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000001000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="alphaf">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000001000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyRgbfButton">
         <property name="toolTip">
          <string>Copy as QColor::fromRgbF</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteRgbfButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>RGBA64</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="red64"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="green64"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="blue64"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="alpha64"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyRgba64Button">
         <property name="toolTip">
          <string>Copy as QColor::fromRgb64</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteRgba64Button">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>CMYK</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cmykC"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cmykM"/>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cmykY"/>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cmykK"/>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cmykA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyCmykButton">
         <property name="toolTip">
          <string>Copy as QColor::fromCmyk</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteCmykButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>CMYKF</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cmykfC"/>
       </item>
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cmykfM"/>
       </item>
       <item>
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cmykfY"/>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cmykfK"/>
       </item>
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cmykfA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyCmykFButton">
         <property name="toolTip">
          <string>Copy as QColor::fromCmykF</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteCmykFButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>HSL</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hslH"/>
       </item>
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hslS"/>
       </item>
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hslL"/>
       </item>
       <item>
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hslA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyHslButton">
         <property name="toolTip">
          <string>Copy as QColor::fromHsl</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHslButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>HSLF</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hslfH"/>
       </item>
       <item>
        <widget class="QLabel" name="label_38">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hslfS"/>
       </item>
       <item>
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hslfL"/>
       </item>
       <item>
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hslfA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyHslFButton">
         <property name="toolTip">
          <string>Copy as QColor::fromHslF</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHslFButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>HSV</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hsvH"/>
       </item>
       <item>
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hsvS"/>
       </item>
       <item>
        <widget class="QLabel" name="label_43">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hsvV"/>
       </item>
       <item>
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hsvA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyHsvButton">
         <property name="toolTip">
          <string>Copy as QColor::fromHsv</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHsvButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>HSVF</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_45">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hsvfH"/>
       </item>
       <item>
        <widget class="QLabel" name="label_46">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hsvfS"/>
       </item>
       <item>
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hsvfV"/>
       </item>
       <item>
        <widget class="QLabel" name="label_48">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hsvfA"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyHsvFButton">
         <property name="toolTip">
          <string>Copy as QColor::fromHsvF</string>
         </property>
         <property name="text">
          <string>Copy QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHsvFButton">
         <property name="toolTip">
          <string>Paste from QColor</string>
         </property>
         <property name="text">
          <string>Paste QColor</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
